src/supervisor.sbvr

```diff
@@ -0,0 +1,461 @@
+Vocabulary: supervisor
+
+-- "(Type)" refers to the target database type to be autogenerated
+
+-- GENERIC TERMS
+
+Term: download progress
+	Concept Type: Integer (Type)
+
+Term: ip address
+	Concept Type: Short Text (Type)
+
+Term: name
+	Concept Type: Short Text (Type)
+
+Term: port
+	Concept Type: Integer (Type)
+
+Term: slug
+	Concept Type: Short Text (Type)
+
+Term: status
+	Concept Type: Short Text (Type)
+
+Term: version
+	Concept Type: Short Text (Type)
+
+Term: uuid
+	Concept Type: Text (Type)
+
+
+
+-- HOST CONFIG PROXY
+-- Discuss: we may not want to incorporate this into the Supervisor model
```

Comment:
I think we should definitely incorporate this in the future, but it might not be needed for a proof of concept.

Reply:
Moved this to a separate file for now that's not used in the proof of concept.